### DISCUSSION TITLE
Tutorial, deploy to Vercel part: Why ship into `main` branch instead of `master`

### DISCUSSION THREAD

**Discussion by artyhedgehog** [vercel/next.js#20923] [Help]
Tutorial in DPS section of [Deploying Your Next.js App page](https://nextjs.org/learn/basics/deploying-nextjs-app/platform-details) says:

> ...
> - Ship: We’ve merged the pull request to `main` to ship to production.
> We strongly recommend using this workflow when developing Next.js apps — it will help you iterate on your app faster.

Is there a reason to use `main` branch for production instead of `master`? I've tried deploying to Vercel with `master` branch and it seems a correct way - judging on Vercel dashboard.
![vercel](https://user-images.githubusercontent.com/7547929/104095515-488b3c80-52a8-11eb-97bb-8e4783a91830.png)

**Comment by timneutkens** [comment]
You can read the reasons here: https://github.com/github/renaming
